design/requirements/_template_requirement.md

```diff
@@ -0,0 +1,18 @@
+# Requirement Title
+The <navigation system> should be able to <shall> <do something>
```

Comment:
I think these angle brackets should be escaped. Markdown allows for some inline markup, so these are not being interpreted correctly by the renderer. They don't show up correctly in Visual Studio code, at least. Searching online, it seems you can use \<navigation system>, for example, instead of <navigation system>.